lib/components/user/sequential-pane-display.js

```diff
@@ -9,59 +11,50 @@ import { SequentialPaneContainer } from './styled'
  */
 class SequentialPaneDisplay extends Component {
   static propTypes = {
-    initialPaneId: PropTypes.string.isRequired,
+    activePaneId: PropTypes.string.isRequired,
     paneSequence: PropTypes.object.isRequired
   }
 
-  constructor (props) {
-    super(props)
-
-    this.state = {
-      activePaneId: props.initialPaneId
-    }
+  /**
+   * Routes to the next pane URL.
+   */
+  _routeTo = nextId => {
+    const { currentPath, activePaneId, routeTo } = this.props
+    routeTo(`${currentPath.replace(new RegExp(`/${activePaneId}$`), `/${nextId}`)}`)
```

Comment:
Are the outermost ` marks needed anymore? Does this need to have a regex? Seems like it could be simplified a bit.

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/280/commits/bdcebdff2bab6237bb804ea49e91e58d44bdb00c to extract the parent path instead.